can_tools/scrapers/official/NH/nh_demographics.py

```diff
@@ -40,6 +43,11 @@ class NHVaccineRace(StateDashboard):
     ]
 
     def fetch(self):
+
+        ts_version = pkg_resources.get_distribution("tableauscraper").version
+        _logger = logging.getLogger(__name__)
+        _logger.warning(f"tableauscraper version {ts_version}")
```

Comment:
using a warning so I know it will show up in the cloud logging and since this will be removed shortly